todos-app/backend/user_service/bin/spring

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# This file loads Spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
+unless defined?(Spring)
```

Comment:
I didn't know about `spring`!
https://github.com/rails/spring

Reply:
I actually didn't know about it either. This was added by Rails when I created a new project 😅

Reply:
oh I should try new version rails..